src/components/cluster/detail/cluster_detail_node_pools_table.js

```diff
@@ -476,21 +467,29 @@ class ClusterDetailNodePoolsTable extends React.Component {
               </GridRowNodePoolsHeaders>
               {nodePools &&
                 nodePools
-                  .sort((a, b) => (a.name > b.name ? 1 : -1))
+                  .sort((a, b) =>
+                    a.name > b.name
```

Comment:
Can you please refactor this to use regular `if` blocks? I know it might be verbose, but it would be much easier to understand

Reply:
Agree